drivers/net/Makefile

```diff
@@ -64,7 +64,6 @@ obj-$(CONFIG_WAN) += wan/
 obj-$(CONFIG_WLAN) += wireless/
 obj-$(CONFIG_WIMAX) += wimax/
 obj-$(CONFIG_IEEE802154) += ieee802154/
-
```

Comment:
Restore this empty line. No need to add changelog noise to other files.

Reply:
Restored